.github/workflows/ci.yaml

```diff
@@ -10,32 +10,28 @@ on:
 jobs:
     cs-fix:
         name: Run code style check
-        runs-on: "ubuntu-20.04"
-        strategy:
-            matrix:
-                php:
-                    - '8.0'
+        runs-on: "ubuntu-22.04"
         steps:
-            - uses: actions/checkout@v2
+            - uses: actions/checkout@v3
 
             - name: Setup PHP Action
               uses: shivammathur/setup-php@v2
               with:
-                  php-version: ${{ matrix.php }}
+                  php-version: '8.0'
```

Comment:
```suggestion
                  php-version: '8.1'
```

Reply:
Can't apply this because one of the dependencies install an older version of `friendsofphp/php-cs-fixer` (v2).

```
composer why friendsofphp/php-cs-fixer
ezsystems/ezplatform-rest       dev-main  requires (for development) friendsofphp/php-cs-fixer (^2.16.0) 
ezsystems/ezplatform-code-style v0.1.0    requires                   friendsofphp/php-cs-fixer (^2.16)   
ezsystems/ezplatform-kernel     1.3.x-dev conflicts                  friendsofphp/php-cs-fixer (3.5.0) 
```

Reply:
Sure, realized that it won't work as soon as CI failed.